modules/ROOT/pages/bolt/message.adoc

```diff
@@ -405,9 +417,7 @@ The `routing::Dictionary(address::String)` field is added in version *4.1* and a
 extra::Dictionary(
   user_agent::String,
   patch_bolt::List<String>,
-  scheme::String,
```

Comment:
`scheme` is still present in the `Messages` table, under `HELLO`. Should it be there?